modules/dhcp_common/subnet.rb

```diff
@@ -48,15 +47,27 @@ def include? ip
     end
 
     def to_s
-      "#{network}/#{netmask}"
+      raise NotImplementedError, "Method 'to_s' must be implemented"
     end
 
-    def cidr
-      IPAddr.new(netmask).to_i.to_s(2).count("1")
+    def prefix
```

Comment:
Please remove these:
 * internal code not meant to be extended
 * you will get an "undefined method" error anyway